content/src/content/jcr_root/apps/core/wcm/sandbox/components/teaser/v1/teaser/_cq_dialog/.content.xml

```diff
@@ -22,14 +22,52 @@
                             sling:resourceType="granite/ui/components/coral/foundation/container"
                             margin="{Boolean}true">
                         <items jcr:primaryType="nt:unstructured">
+                            <withCTA
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                    fieldDescription="When checked, allow adding Call To Action (CTA) elements."
```

Comment:
I'd reformulate as follows: 
Check to allow adding Call To Action (CTA) elements.